Find a functionally equivalent code:android.util.EventLog.Event.getTag:COMMENT
Method Modifier: public      
Comment:/**
 * @return the type tag code of the entry
 */

@@ -1,3 +1,7 @@
 {
-    return mBuffer.getInt(TAG_OFFSET);
+    int offset = mBuffer.getShort(HEADER_SIZE_OFFSET);
+    if (offset == 0) {
+        offset = V1_PAYLOAD_START;
+    }
+    return mBuffer.getInt(offset);
 }

